mlir/include/mlir/Dialect/LLVMIR/NVVMOps.td

```diff
@@ -462,24 +462,28 @@ def NVVM_MBarrierTestWaitSharedOp : NVVM_Op<"mbarrier.test.wait.shared">,
 // NVVM synchronization op definitions
 //===----------------------------------------------------------------------===//
 
-def NVVM_Barrier0Op : NVVM_IntrOp<"barrier0"> {
+def NVVM_Barrier0Op : NVVM_Op<"barrier0"> {
```

Comment:
do you need to change NVVM_IntrOp->NVVM_Op?

Reply:
Yep. Otherwise, references to the no-longer present Intrinsic::barrier0 get generated by this record such as in `NVVMConvertibleLLVMIRIntrinsics.inc`